TransformationSystem/Client/TaskManager.py

```diff
@@ -499,6 +611,41 @@ def getOutputData( self, paramDict, moduleLocation ):
     return module.execute()
 
   def submitTransformationTasks( self, taskDict ):
+
+    if 'BulkJobObject' in taskDict:
+      result = self.__submitTransformationTasksBulk( taskDict )
+    else:
+      result = self.__submitTransformationTasks( taskDict )
+    return result
+
+  def __submitTransformationTasksBulk( self, taskDict ):
+    """ Submit jobs one by one
+    """
+    startTime = time.time()
+    transID = taskDict[taskDict.keys()[0]]['TransformationID']
+    oJob = taskDict.pop( 'BulkJobObject' )
+    if oJob is None:
+      self._logError( 'submitTransformationTasksBulk: no bulk Job object found', transID = transID )
+      return S_ERROR( ETSUNK, 'No bulk job object provided for submission' )
+
+    result = self.submitTaskToExternal( oJob )
+    if not result['OK']:
+      return result
+
+    jobIDList = result['Value']
+    ind = 0
+    for taskID in sorted( taskDict ):
```

Comment:
What is important here is that tasks are processed in the same order as in __prepareTransformationTasksBulk() above